MonoGame.Framework.Content.Pipeline/Graphics/Font/CharacterRegion.cs

```diff
@@ -4,23 +4,23 @@
 
 namespace Microsoft.Xna.Framework.Content.Pipeline.Graphics
 {
-	// Describes a range of consecutive characters that should be included in the font.
-	[TypeConverter(typeof(CharacterRegionTypeConverter))]
-	public struct CharacterRegion
-	{
-	    public char Start;
-	    public char End;
+    // Describes a range of consecutive characters that should be included in the font.
+    [TypeConverter(typeof(CharacterRegionTypeConverter))]
+    public struct CharacterRegion
+    {
+        public char Start;
+        public char End;
 
-		// Enumerates all characters within the region.        
-	    public IEnumerable<Char> Characters()
-	    {
-	        for (var c = Start; c <= End; c++)
-	        {
-	            yield return c;
-	        }
-	    }
+        // Enumerates all characters within the region.
+        public IEnumerable<char> Characters()
+        {
+            for (var c = Start; c <= End; c++)
+            {
+                yield return c;
+            }
+        }
 
-	    // Constructor.
+        // Constructor.
         public CharacterRegion(char start, char end)
         {
             if (start > end)
```

Comment:
Maybe consider updating the exception to include a clear message. For example:

    if (start > end)
        throw new ArgumentException($"The start character '{start}' must not be greater than the end character '{end}'.");

As well as a unit test.

Reply:
Exception message updated. Thanks.